### PR TITLE
Main file for testing reparam quckly + outline for reparam phase I

### PR DESCRIPTION
**Pull request by EricMFischer**
Main file for testing reparam quckly + outline for reparam phase I

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import cv2
```
Comment (allen-n) [inline main.py:1]: What is the point of copying all of this into a .py file? It's more useful in the notebook for testing anyways.

Comment (EricMFischer) [reply main.py:1]: It runs faster in a python file

File: tsnake/grid.py [hunk 2]
```diff
@@ -322,22 +322,39 @@ def get_snake_intersections(self, snakes: TSnakes) -> [[Point]]:
...
 
-    def reparameterize(self, snakes: TSnakes):
+    def reparameterize_phase_one(self, snakes: TSnakes):
         '''
-        Takes a list of T-Snake instances and returns a new list of TSnakes
-        (which have possibly been split/merged/etc).
-        If no snakes are split/merged, then:
-        1) the snakes are presented in the same order as before, AND
-        2) each snake's nodes are presented in the same order.
+        Takes a list of T-Snakes and returns a new list of T-Snakes (which have possibly been split/merged/etc).
```
Comment (allen-n) [inline tsnake/grid.py:327]: This isn't an outline, it's just re-typing more or less the exact text of the paper. All for it if it helps you understand what to do, but it's really time for the rubber to hit the road here, I've actually written the functional skeleton for this in the meantime on my end.

**Review by allen-n — APPROVED** [review]
This all looks fine, but the next commit *really* has to be code, its too late to still be trying to understand what's going on here, we've gone through it in depth numerous times.

**Pull request merged**